modules/machineset-manually-scaling.adoc

```diff
@@ -36,7 +36,26 @@ $ oc edit machineset <machineset> -n openshift-machine-api
 You can scale the MachineSet up or down. It takes several minutes for the new
 machines to be available.
 +
+
+= The MachineSet delete policy
+`Random`, `Newest`, and `Oldest` are the three supported options. The default is `Random`, meaning  that random machines are chosen and deleted when scaling MachineSets down. The delete policy can be set according to the use case by modifying the particular MachineSet:
+
+----
+
+spec:
+  deletePolicy: <delete policy>
+  replicas: <desired replica count>
+
+----
+
+Specific machines can also be priortized for deletion by adding the annotation `machine.openshift.io/cluster-api-delete-machine` to the machine of interest, regardless of the delete policy.
+
 [IMPORTANT]
 ====
 By default, the {product-title} router pods are deployed on workers. Because the router is required to access some cluster resources, including the web console, do not scale the worker MachineSet to `0` unless you first relocate the router pods.
 ====
+
+[NOTE]
+====
+Custom/Pet MachineSets can be used for use cases requiring that services run on specific nodes and that those services are ignored by the controller when the MachineSets are scaling down. This prevents service disruption.
```

Comment:
I don't understand this statement? there's no such a thing like a "pet" machineSet. Any machineSet consider machines "cattle". Your workloads availability is ensured via your pod disruption budget.

Reply:
Pet is not a good term.   Custom is probably better.   The use case is when you require a class of cloud instances which are different from your cattle.   e.g. My cattle need 2 vCPU and 8Gi, so the majority of my MachineSets use that instance type.    To get a different class of cloud instan ces for my Elastic clusters (8 vCPU, 32Gi), I need a machineset that spins up m5.2xlarge.   AFAIK the only way to address this use case today is with custom machinesets.

Reply:
@chaitanyaenr Once you update the language here, we can get this moved forward. Thanks!

Reply:
@ahardin-rh updated, thanks.